index-tr.html

```diff
@@ -109,6 +109,8 @@ <h2>Daha Az Kısa Versiyon</h2>
 <p><small><em>Orijinal kaynak: <a href="http://2012.jsconf.us/#/about">http://2012.jsconf.us/#/about</a> &amp; <a href="http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy">The Ada Initiative</a><br>
 
 Çeviri: <a href="http://ahmetalpbalkan.com">Ahmet Alp Balkan</a><br>
+  
+Çeviri geliştirme desteği: <a href="https://tr.linkedin.com/in/symutlubay">Sarper Yasin MUTLUBAY</a><br>
```

Comment:
Oh, thanks for outlining the fact that the Turkish translation was signed!
From what I can see, this is the only one. It could be interesting to consider it to ensure people feel more committed to maintaining translations.
However, for the moment, I think it is better if we avoid adding an incentive to self-advertise in translations in order to minimise the amount of contributions that would be made mostly to increase SEO, and to avoid having an ever growing list of contributors…

I feel it would be better to remove this line as well as the one above. What do you think of it? 🙂

Reply:
done 👍  :)